### PR TITLE
Resprites shipkey (random colors) and make wearable on neck

### PR DESCRIPTION
**Pull request by mel-byond**
## About The Pull Request

Resprites shipkey so it isn't SO DAMN HARD TO CLICK AND FIND WHEN IT BLENDS INTO THE HUD aAAAAAAAAA

- [x] Resprite shipkey
- [x] Randomize the plastic body color, so the ships atleast have differing colors
- [x] Make neck wearable

## Why It's Good For The Game

yeah i kinda kept losing my keys in my inventory due to the sprite being so small :((

## Changelog
:cl:
qol: Resprites ship key and makes it wearable on the neck by a fancy ball chain!
/:cl:

### CODE REVIEW

**Comment by mel-byond** [comment]
Apparently there is no worn icon for it so far

File: code/modules/overmap/ships/controlled_ship_datum.dm [hunk 1]
```diff
@@ -271,14 +271,42 @@
...
+		var/our_color = pick(key_colors)
+		add_atom_colour(key_colors[our_color], FIXED_COLOUR_PRIORITY)
+		update_icon()
 	name = "ship key ([master_ship.name])"
 
+/obj/item/key/ship/update_overlays()
+	. = ..()
+	if(!random_color) //icon override
+		return
+	var/mutable_appearance/base_overlay = mutable_appearance(icon, "shipkey_metalybits")
```
Comment (ZephyrTFA) [inline code/modules/overmap/ships/controlled_ship_datum.dm:306]: If this overlay never changes, make it static

Comment (mel-byond) [reply code/modules/overmap/ships/controlled_ship_datum.dm:306]: Does this still require a change or nah? Seeing that this is resolved now
The code itself should be the same as the screwdriver

**Comment by mel-byond** [comment]
Alright, I am back, I should have more time now. Is this alright?

**Review by ZephyrTFA — APPROVED** [review]

**Pull request merged**